src/modules/micrortps_bridge/micrortps_client/microRTPS_client_main.cpp

```diff
@@ -153,7 +174,7 @@ static int micrortps_start(int argc, char *argv[])
 						       _options.sw_flow_control, _options.hw_flow_control, _options.verbose_debug);
 			PX4_INFO("UART transport: device: %s; baudrate: %" PRIu32 "; sleep: %" PRIu32 "ms; poll: %" PRIu32
 				 "ms; flow_control: %s",
-				 _options.device, _options.baudrate, _options.sleep_ms, _options.poll_ms,
+				 _options.device, _options.baudrate, _options.sleep_us, _options.poll_ms,
```

Comment:
The string says `ms` but it's now `us`, right?

Reply:
Typo. To fix